docs/troubleshooting.md

```diff
@@ -35,3 +36,28 @@ using Minishift.
   systemctl start virtlogd
   systemctl enable virtlogd
   ```
+
+
+## XHYVE drivers
+
+- Could not create vmnet interface, permission denied or no entitlement
+
```

Comment:
We need some context here, something like, Minishift fails to start because of `the error mssg` will keep it consistent with the above issues, but whatever is the case here needs to be added.